### PR TITLE
Feat/55: Support configuration in pyproject.toml

### LINKED ISSUE #55: Support pyproject.toml configuration

### PR DESCRIPTION
**Pull request by EdgyEdgemond**
closes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,21 @@ release = true
 allow_dirty = false
 ```
 
+Example `pyproject.toml`:
+
+```ini
```
Comment (myddrin) [inline README.md:70]: Should that be `toml`?

File: README.md [hunk 2]
```diff
@@ -110,9 +125,9 @@ General configuration is grouped in a `[changelog_gen]` section.
 
   Example:
 
-```ini
-[changelog_gen]
-issue_link = http://github.com/EdgyEdgemond/changelog-gen/issues/{issue_ref}
+```toml
+[toolchangelog_gen]
```
Comment (myddrin) [inline README.md:129]: typo: `tool.changelog_gen`

**Review by myddrin — COMMENTED** [review]
I still need to review unit tests.
Should we keep the examples for the setup.cfg or is your goal to remove support for it over time?

File: changelog_gen/extractor.py [hunk 3]
```diff
@@ -84,8 +84,7 @@ def extract_version_tag(sections: SectionDict) -> str:
     """
     version_info_ = BumpVersion.get_version_info("patch")
     current = version_info_["current"]
-    # TODO(edgy): In the case of 0.x releases, everything should be one semver lower.
-    # https://github.com/EdgyEdgemond/changelog-gen/issues/50
+
```
[the remark this answers is not in the document]
Comment (myddrin) [reply changelog_gen/extractor.py:87]: Ignore this - I did not realise there are several other MR open!

**Comment by EdgyEdgemond** [comment]
Ideal would be to drop setup.cfg down the line, anyone currently using setup.cfg has the examples they need at hand. Anyone coming along, it is cleaner and simpler to direct them to pyproject.toml

File: pyproject.toml [hunk 4]
```diff
@@ -29,6 +30,20 @@ pytest-git = "^1.7.0"
...
 
+[tool.changelog_gen]
+release = true
+commit = true
+allowed_branches = [
+    "master",
+]
+issue_link = "https://github.com/EdgyEdgemond/changelog-gen/issues/{issue_ref}"
+
+[tool.changelog_gen.sesion_mapping]
```
Comment (EdgyEdgemond) [inline pyproject.toml:41]: typo fixed in a later PR, when mapping feature gets expanded

**Review by myddrin — APPROVED** [review]

**Pull request merged**